twitterscraper/tweet.py

```diff
@@ -17,27 +17,34 @@ def __init__(self, user, fullname, id, url, timestamp, text, replies, retweets,
         self.retweets = retweets
         self.likes = likes
         self.html = html
+        self.mentions = mentions
 
     @classmethod
     def from_soup(cls, tweet):
+        mentions = []
```

Comment:
This looks a bit cumbersome. 
Would it work if you implemented `mentions = tweet.find('div', 'tweet')['data-mentions'] or ""` on line 47, and subsequently `self.mentions = mentions.split()` on line 20. 
In this case, if the passed argument is an empty string, the applied split() operation will results in an empty list.